rerun_py/rerun_sdk/rerun/_rerun2/datatypes/_overrides/annotation_context.py

```diff
@@ -29,7 +29,8 @@
 
 def classdescription_init(
     self: ClassDescription,
-    info: AnnotationInfoLike,
+    info: AnnotationInfoLike = 0,
```

Comment:
This still strikes me as a bit of a footgun, but it matches the pre-existing behavior.

Reply:
I'm gonna remove it in a follow-up-PR, but keep it here to not break API further in a patch release

Reply:
If you want to break it in the future, why not break it now? Is there a meaningful difference between breaking it in 0.8 and breaking it in 0.9?

Reply:
It's more that I want to do the least intrusive change for 0.8.1.

If we require `info` to be explicit, should we also make it explicit for `log_points`? Why require explicitness in one but not the other?

Reply:
Fair enough.